mobile/src/main/java/ch/epfl/sdp/mobile/ui/features/social/SocialScreen.kt

```diff
@@ -0,0 +1,81 @@
+package ch.epfl.sdp.mobile.ui.features.social
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import ch.epfl.sdp.mobile.ui.features.ProfileColor
+import ch.epfl.sdp.mobile.ui.i18n.LocalLocalizedStrings
+
+/**
+ * This screen display all register user of the app
+ *
+ * @param state the [FollowingState], manage the composable contents
+ * @param modifier the [Modifier] for the composable
+ */
+@Composable
+fun SocialScreen(state: FollowingState, modifier: Modifier = Modifier) {
+
+  val strings = LocalLocalizedStrings.current
+  val friendList = remember { state.players }
+  Column {
+    Text(
+        text = strings.SocialFollowingTitle,
+        color = MaterialTheme.colors.primary,
+        style = MaterialTheme.typography.h4,
+        modifier = Modifier.padding(16.dp))
+    LazyColumn(modifier.testTag("friendList")) {
```

Comment:
Bug : the `modifier` argument should be applied to `Column`, not `LazyColumn`.
```suggestion
  Column(modifier) {
    Text(
        text = strings.SocialFollowingTitle,
        color = MaterialTheme.colors.primary,
        style = MaterialTheme.typography.h4,
        modifier = Modifier.padding(16.dp))
    LazyColumn(Modifier.testTag("friendList")) {
```